File: Ass6/Ass6P1.c
#include<stdio.h>

void Number(int iNo)
{
    if(iNo < 50)
    {
        printf("Less");
    }
    else if((iNo >= 50) && (iNo <= 100))
    {
        printf("Medium");     
    }
    else
    {
        printf("Large");
    }
}
int main()
{
    int ivalue = 0;
    
    printf("Enter the number");
    scanf("%d",&ivalue);

    Number(ivalue);

    
    return 0;
}

File: Ass6/Ass6P1.java

import java.util.Scanner;

class Number
{
    public void Num(int iNo)
    {
        if( iNo < 50)
        {
            System.out.println(iNo+"Small");
        }
        else if((iNo > 50) && (iNo < 100))
        {
            System.out.println(iNo+"Medium");

        }
        else
        {
            System.out.println(iNo+"Large");
        }
    }
}

class Ass6P1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int ivalue = 0;

        System.out.println("Enter the Number :- ");
        ivalue = sobj.nextInt();

        Number nobj=new Number();

        nobj.Num(ivalue);


    }
}